frontend/public/components/pod.jsx

```diff
@@ -122,12 +122,12 @@ const ContainerTable = ({heading, containers, pod}) => <div className="co-m-pane
     <div className="co-m-table-grid co-m-table-grid--bordered">
       <div className="row co-m-table-grid__head">
         <div className="col-sm-2 col-xs-4">Name</div>
-        <div className="col-md-2 col-sm-3 hidden-xs">Id</div>
         <div className="col-md-2 col-sm-3 col-xs-8">Image</div>
         <div className="col-md-1 col-sm-2 hidden-xs">State</div>
         <div className="col-md-1 col-sm-2 hidden-xs">Restarts</div>
         <div className="col-md-2 hidden-sm hidden-xs">Started</div>
         <div className="col-md-2 hidden-sm hidden-xs">Finished</div>
+        <div className="col-md-1 col-sm-2 hidden-xs">Exit Code</div>
```

Comment:
We should look at expanding one of the other columns so we use all 12 grid columns.